src/test/webServer.ts

```diff
@@ -85,19 +84,17 @@ class LaunchServer {
   }
 
   private async _waitForProcess() {
-    if (this.config.waitForPort) {
-      await this._waitForAvailability(this.config.waitForPort);
-      const baseURL = `http://localhost:${this.config.waitForPort}`;
-      process.env.PLAYWRIGHT_TEST_BASE_URL = baseURL;
-      console.log(`Using baseURL '${baseURL}' from config.launch.`);
-    }
+    await this._waitForAvailability();
+    const baseURL = `http://localhost:${this.config.port}`;
+    process.env.PLAYWRIGHT_TEST_BASE_URL = baseURL;
+    console.log(`Using baseURL '${baseURL}' from config.launch.`);
```

Comment:
Let's not log here?
```suggestion
```

Reply:
Either not log or change it to: `console.log(`Using baseURL '${baseURL}' from config.webServer.`);` Both fine for me.

Reply:
No log please.